libkbfs/block_retrieval_queue.go

```diff
@@ -213,6 +215,8 @@ func (brq *blockRetrievalQueue) notifyWorker(priority int) {
 func (brq *blockRetrievalQueue) CacheAndPrefetch(ctx context.Context,
 	ptr BlockPointer, block Block, kmd KeyMetadata, priority int,
 	lifetime BlockCacheLifetime, hasPrefetched bool) (err error) {
+	didUpdateCh := make(chan struct{})
+	donePrefetch := hasPrefetched
```

Comment:
I can do s/hasPrefetched/triggeredPrefetch everywhere except in the main struct, since users' caches will already have `HasPrefetched` encoded into their disk cache metadata. WDYT?